doc/architecture/software-update.rst

```diff
@@ -35,7 +63,7 @@ and consumed by the software update client running on supported devices.
 
 Update information is secured by generating hashes of each file and then
 signing the file containing the references to the hashes.
-A secure channel is not required to distribute the updates.
+There is no need for a secure channel, for distributing the updates.
```

Comment:
swupd client is indeed not using hash and signature checks to validate the received data. What Igor describes here is the long term goal, not the current implementation.

I don't know what the document is meant to describe (both?), so I don't have a specific proposal for a replacement. Given that people will read this document to understand how Ostro OS currently works, at least the current status should be mentioned.